website/docs/r/ses_domain_identity_verification.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "aws"
+page_title: "AWS: ses_domain_identity_verification"
+sidebar_current: "docs-aws-resource-ses-domain-identity-verification"
+description: |-
+  Waits for and checks successful verification of an SES domain identity.
+---
+
+# aws_ses_domain_identity_verification
+
+Represents a successful verification of an SES domain identity.
+
+Most commonly, this resource is used together with [`aws_route53_record`](route53_record.html) and
+[`aws_ses_domain_identity`](ses_domain_identity.html) to request an SES domain identity,
+deploy the required DNS verification records, and wait for verification to complete.
+
+~> **WARNING:** This resource implements a part of the verification workflow. It does not represent a real-world entity in AWS, therefore changing or deleting this resource on its own has no immediate effect.
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `domain` - (Required) The domain name of the SES domain identity to verify.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - The domain name of the domain identity.
+* `arn` - The ARN of the domain identity.
+
+## Timeouts
+
+`acm_ses_domain_identity_verification` provides the following [Timeouts](/docs/configuration/resources.html#timeouts)
+configuration options:
+
+- `create` - (Default `45m`) How long to wait for a domain identity to be verified.
+
+## Example Usage
```

Comment:
Nitpick: usually the Example Usage is above the arguments (one day we'll have automated documentation 😄 )